internal/tiltfile/api.go

```diff
@@ -0,0 +1,41 @@
+package tiltfile
+
+import (
+	"embed"
+	"io/fs"
+	"os"
+	"path/filepath"
+)
+
+//go:embed api/*.py api/config/*.py api/os/*.py api/shlex/*.py api/sys/*.py api/v1alpha1/*.py
```

Comment:
```suggestion
//go:embed api/*.py api/*/*.py
```

Reply:
if only to make it less likely that someone adds a module and is confused when / doesn't notice that there are no docs